src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -550,6 +550,20 @@ public static void CopyTo_Invalid()
         [InlineData("Goodbye", 0, "Hello", 0, 5, StringComparison.OrdinalIgnoreCase, -1)]
         [InlineData("HELLO", 2, "hello", 2, 3, StringComparison.OrdinalIgnoreCase, 0)]
         [InlineData("Hello", 2, "Goodbye", 2, 3, StringComparison.OrdinalIgnoreCase, -1)]
+        [InlineData("A", 0, "x", 0, 1, StringComparison.OrdinalIgnoreCase, -1)]
```

Comment:
It would be useful to do run all API in the test for both argument orders ie do all tests as both `string.Compare(a,b)` and `string.Compare(b,a)`. It is done for the CompareTo case, but not others.